### PR TITLE
masterclass: Including link to debugging masterclass

### PR DESCRIPTION
**Pull request by garethtdavies**
Adding link to debugging masterclass and cleaning some formatting

Change-type: patch
Signed-off-by: Gareth Davies <gareth@balena.io>

### CODE REVIEW

**Comment by garethtdavies** [comment]
I'll get back to #1295 soon as well so we have this info directly in the docs (and searchable)

File: pages/learn/more/masterclass.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ Feedback on the masterclasses are welcome and should be given by creating an iss
 
 Currently, the following masterclasses are available:
 
-* [{{ $names.cli }} masterclass]({{ $links.githubMain }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.cli }}.
-* [{{ $names.cli }} advanced masterclass]({{ $links.githubMain }}/balena-cli-advanced-masterclass) - a guide to the advanced features available in {{ $names.cli }}.
+* [{{ $names.company.upper }} CLI masterclass]({{ $links.githubMain }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.cli }}.
```
Comment (xginn8) [inline pages/learn/more/masterclass.md]: Why the change here?

Comment (garethtdavies) [reply pages/learn/more/masterclass.md]: Because it's lowercase and there is no .upper and .lower - though I should probably just add them.  I'll update this.

Comment (xginn8) [reply pages/learn/more/masterclass.md]: Makes sense then, thanks for clarifying!

Comment (garethtdavies) [reply pages/learn/more/masterclass.md]: I've pushed an update and updated all existing references
